pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneStarter.java

```diff
@@ -149,7 +149,7 @@ void start() throws Exception {
         if (!onlyBroker) {
             // Start LocalBookKeeper
             bkEnsemble = new LocalBookkeeperEnsemble(numOfBk, zkPort, bkPort, zkDir, bkDir, wipeData);
-            bkEnsemble.start();
+            bkEnsemble.startStandalone();
```

Comment:
many other tests uses `LocalBookkeeperEnsemble.start()` so, should we change in all other tests as well?